x/auth/vesting/client/testutil/suite.go

```diff
@@ -130,3 +130,233 @@ func (s *IntegrationTestSuite) TestNewMsgCreateVestingAccountCmd() {
+				fmt.Sprintf("--%s=%s", cli.FlagVesting, "testdata/badjson"),
+			},
+			expectErr: true,
+		},
+		{
+			name: "bad vesting periods",
+			args: []string{
+				sdk.AccAddress("addr13______________").String(),
+				fmt.Sprintf("--%s=%s", cli.FlagVesting, "testdata/badperiod.json"),
+			},
+			expectErr: true,
+		},
+	} {
+		s.Run(tc.name, func() {
+			clientCtx := val.ClientCtx
+
+			bw, err := clitestutil.ExecTestCLICmd(clientCtx, cli.NewMsgCreateClawbackVestingAccountCmd(), tc.args)
+			if tc.expectErr {
+				s.Require().Error(err)
+			} else {
+				s.Require().NoError(err)
+				s.Require().NoError(clientCtx.Codec.UnmarshalJSON(bw.Bytes(), tc.respType), bw.String())
+
+				txResp := tc.respType.(*sdk.TxResponse)
+				s.Require().Equal(tc.expectedCode, txResp.Code)
+			}
+		})
+	}
+}
+
+func (s *IntegrationTestSuite) TestNewMsgClawbackCmd() {
+	val := s.network.Validators[0]
+	addr := sdk.AccAddress("addr30______________")
+
+	_, err := clitestutil.ExecTestCLICmd(val.ClientCtx, cli.NewMsgCreateClawbackVestingAccountCmd(), []string{
+		addr.String(),
+		fmt.Sprintf("--%s=%s", flags.FlagFrom, val.Address),
+		fmt.Sprintf("--%s=%s", cli.FlagLockup, "testdata/periods1.json"),
+		fmt.Sprintf("--%s=%s", cli.FlagVesting, "testdata/periods1.json"),
+		fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+		fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+	})
+	s.Require().NoError(err)
+
+	for _, tc := range []struct {
+		name         string
+		args         []string
+		expectErr    bool
+		expectedCode uint32
+		respType     proto.Message
+	}{
+		{
+			name: "basic",
+			args: []string{
+				addr.String(),
+				fmt.Sprintf("--%s=%s", flags.FlagFrom, val.Address),
+				fmt.Sprintf("--%s=%s", cli.FlagDest, sdk.AccAddress("addr32______________").String()),
+				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+				fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+				fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+			},
+			expectErr:    false,
+			expectedCode: 0,
+			respType:     &sdk.TxResponse{},
+		},
+		{
+			name: "bad vesting addr",
+			args: []string{
+				"foo",
+			},
+			expectErr: true,
+		},
+		{
+			name: "bad dest addr",
+			args: []string{
+				addr.String(),
+				fmt.Sprintf("--%s=%s", cli.FlagDest, "bar"),
+			},
+			expectErr: true,
+		},
+		{
+			name: "default dest",
+			args: []string{
+				addr.String(),
+				fmt.Sprintf("--%s=%s", flags.FlagFrom, val.Address),
+				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+				fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+				fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+			},
+			expectErr:    false,
+			expectedCode: 0,
+			respType:     &sdk.TxResponse{},
+		},
+	} {
```

Comment:
Why is the test execution block wrapped in parentheses?

Reply:
`for` is wrapped around testcases itself!